source/features/latest-tag-button.tsx

```diff
@@ -139,7 +134,7 @@ async function init(): Promise<false | void> {
 		link.setAttribute('aria-label', 'Visit the latest release');
 	}
 
-	link.classList.add('tooltipped', 'tooltipped-ne');
+	link.classList.add('tooltipped', 'tooltipped-ne', 'rgh-latest-tag-button');
```

Comment:
This change isn't related to bugfix but rather #3356. With current changes it is really inconvenient to find a succinct selector to customize the button's size, so I think adding a class won't hurt.

Reply:
I should have added it in this line as well 😅 https://github.com/sindresorhus/refined-github/pull/3497/files#diff-5664f505bec7f184c9588cc203936104R107